src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/StoredDocumentHalResource.java

```diff
@@ -62,10 +61,9 @@
     @JsonProperty("ttl")
     private Date ttl = null;
 
-    @JsonInclude(JsonInclude.Include.NON_NULL)
     public void addLinks(UUID documentId) {
         add(linkTo(methodOn(CaseDocumentAmController.class).getDocumentbyDocumentId("dsds", documentId, "323", "caseworker-1")).withSelfRel());
-
+        add(linkTo(methodOn(CaseDocumentAmController.class).getDocumentBinaryContentbyDocumentId("dsds", documentId, "323", "caseworker-1")).withSelfRel());
```

Comment:
why are we adding some dummy values for generating links? Are the links generating correctly?

Reply:
We are extracting the path at method level through Links to  Controller Methods Hateoas API which required to pass parameters as per method signature define in  controller level.Off course, links are generated correctly.